packages/server/src/api.ts

```diff
@@ -9,6 +9,8 @@ import { MessageApi } from './messages/api'
 import { SocketManager } from './socket/manager'
 import { SyncApi } from './sync/api'
 import { UserApi } from './users/api'
+import * as Sentry from '@sentry/node'
```

Comment:
Make sure that ESLint and Prettier run on your code. This should be automatic when using VSCode.